docs/Architecture Decision Record/011-buildpacks.md

```diff
@@ -33,3 +33,50 @@ Our recommendation is to move to Cloud.gov buildpacks at this time. They are alr
 **Notes**
 - Docker containers will still need to be maintained for local development and CI/CD
 - Docker containers will still need to be hardened for CI/CD
+
+## Restaging for updated buildpacks
+
+As described in #1045, cloud.gov will inform us that buildpack(s) we use have been updated to a newer version via e-mail to all users with 'developer' role. The e-mail provides specific CloudFoundry CLI steps needed but we have already captured our deployment strategy process/commands in scripts/deploy-backend.sh. Running that script is the preferred methodology. Presently, the e-mail does not provide any specifics about the update, just that there was an update.
+
+Below is the restaging process in full:
+    1. Upon receipt of email from cloud.gov, restage against dev:
+```bash
+user@host$ cf login -a api.fr.cloud.gov --sso
+API endpoint: api.fr.cloud.gov
+
+Temporary Authentication Code ( Get one at https://login.fr.cloud.gov/passcode ): 
+Authenticating...
+OK
+
+
+Targeted org hhs-acf-prototyping.
+
+Select a space:
+1. tanf-dev
+2. tanf-staging
+
+Space (enter to skip): 1
+Targeted space tanf-dev.
+
+API endpoint:   https://api.fr.cloud.gov
+API version:    3.101.0
+user:           abottoms@goraft.tech
+org:            hhs-acf-prototyping
+space:          tanf-dev 
+$ bash scripts/deploy-backend.sh rolling tdp-backend
```

Comment:
```suggestion
$ cf restage tdp-backend-a11y
$ cf restage tdp-backend-raft
$ cf restage tdp-backend-qasp
$ cf restage tdp-backend-sandbox
```
Using the restage here prevents overwriting code currently deployed with what the user has in their local clone.